.circleci/config.yml

```diff
@@ -1,21 +1,45 @@
 version: 2.1
-jobs:
-  build:
-    docker:
-      - image: cimg/node:18.7.0
+
+defaults: &defaults
+  docker:
+    - image: cimg/node:18.7.0
+
+commands:
+  setup:
     steps:
       - checkout
       - run: node --version
       - run: sudo apt-get -y update
       - run: sudo apt-get -y install libudev-dev libusb-1.0-0-dev
       - run: yarn install
-      - run: yarn eslint
+
+jobs:
+  audit:
+    <<: *defaults
+    steps:
+      - setup
+      - run: yarn audit
```

Comment:
If you ever want to ignore some vulnerability (e.g. because there's just not a patch available yet) replacing raw `yarn audit` with [improved-yarn-audit](https://www.npmjs.com/package/improved-yarn-audit) may come in handy